docs/src/components/InstallScripts.tsx

```diff
@@ -4,35 +4,38 @@ import { Tabs, TabItem } from '@aws-amplify/ui-react';
 type WebFramework = 'react' | 'vue' | 'angular';
 
 interface TerminalCommandProps {
-  framework: WebFramework;
-  packageManager: 'npm' | 'yarn';
+  framework?: WebFramework;
+  packageManager?: 'npm' | 'yarn';
+  terminalCommand?: string;
 }
 
-const TerminalCommand = ({
+const frameworkInstallScript = (framework, packageManager) =>
+  `${
+    packageManager === 'npm' ? 'npm i' : 'yarn add'
+  } @aws-amplify/ui-${framework} aws-amplify`;
+
+export const TerminalCommand = ({
   framework,
   packageManager,
+  terminalCommand,
 }: TerminalCommandProps) => {
-  const frameworkInstallScript = `${
-    packageManager === 'npm' ? 'npm i' : 'yarn add'
-  } @aws-amplify/ui-${framework} aws-amplify`;
+  const command = terminalCommand
+    ? terminalCommand
+    : frameworkInstallScript(framework, packageManager);
 
   return (
     <code className="install-code__container">
-      <p className="install-code__content">{frameworkInstallScript}</p>
+      <p className="install-code__content">{command}</p>
       <CopyButton
         className="install-code__button"
-        copyText={frameworkInstallScript}
+        copyText={command}
         variation="link"
       />
     </code>
   );
 };
 
-interface InstallScriptsProps {
-  framework: WebFramework;
-}
-
-export const InstallScripts = ({ framework }: InstallScriptsProps) => {
+export const InstallScripts = (framework: WebFramework) => {
```

Comment:
`framework` needs to be destructured here since it's a prop of `InstallScripts`. Take a look at the installation command on the page localhost:5000/react/getting-started/installation.

Reply:
Ah good catch. Thanks!